README.md

```diff
@@ -11,6 +11,7 @@ This tool uses [`subxt`](https://github.com/paritytech/subxt) to connect to an R
 ### Example
 
 ```bash
+cargo build --release
```

Comment:
Even with `default-run`, you still have to run `cargo build` manually with multiple binaries. See https://github.com/rust-lang/cargo/issues/7387.